lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestCloseTask.cs

```diff
@@ -322,6 +388,92 @@ public void OnError(Exception error)
             }
         }
 
+        private sealed class BreakTestTask : ITask, IObserver<ICloseEvent>
```

Comment:
Can you add documentations to all these tasks. Its very difficult to see clearly what each one is trying to do. Also can you expand on the names of tasks to make them more informative?